src/Autofac/Core/Resolving/ResolveOperationBase.cs

```diff
@@ -83,48 +73,43 @@ protected ResolveOperationBase(ISharingLifetimeScope mostNestedLifetimeScope, IR
         }
 
         /// <summary>
-        /// Gets the active resolve request.
+        /// Gets or sets the active resolve request.
         /// </summary>
-        public ResolveRequestContextBase? ActiveRequestContext { get; private set; }
```

Comment:
Could be useful for any code that only has access to the operation to know which Resolve is currently running. I'm picturing more monitoring/diagnostic extensions as much as anything else.